iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,69 @@
+// MARK: - Stack
+
+/// Provides standard interface for stack data structure organized according to `LIFO` principle.
```

Comment:
Please wrap doc comments at 80th character as max (more details at https://softwareengineering.stackexchange.com/questions/604/is-the-80-character-limit-still-relevant-in-times-of-widescreen-monitors)

```suggestion
/// Provides standard interface for stack data structure
/// organized according to `LIFO` principle.
```

Reply:
Otherwise it's not convenient to review long lines:

<img width="544" alt="Screen Shot 2020-09-26 at 08 49 58" src="https://user-images.githubusercontent.com/6073510/94331469-49d47a00-ffd5-11ea-9a6f-7052d8392953.png">

Reply:
Fixed.